fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/services/CreateResourceService.java

```diff
@@ -35,30 +35,34 @@
      * Create a new NonRdfSource resource.
      *
      * @param txId The transaction ID for the request.
+     * @param userPrincipal principal of user perform service
      * @param fedoraId The internal identifier of the parent.
      * @param slug The Slug header or null if none.
      * @param isContained The new resource is contained by fedoraId (ie. POST).
      * @param contentType The content-type header or null if none.
+     * @param filename original filename of the binary
```

Comment:
For consistency:  "The original file name of the binary."
and "The size of the content stream."